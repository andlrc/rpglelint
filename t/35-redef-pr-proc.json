[
   {
      "column":7,
      "file":"t/35-redef-pr-proc.rpgle",
      "line":4,
      "message":"redefinition of 'myint' as a different kind of symbol",
      "severity":"warning"
   },
   {
      "column":7,
      "file":"t/35-redef-pr-proc.rpgle",
      "line":3,
      "message":"previous definition is here",
      "severity":"note"
   }
]
