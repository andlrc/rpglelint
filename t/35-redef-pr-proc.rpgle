// rpglelint: -Wredefining-symbol

dcl-s myint int(10);
dcl-s myint int(10);

dcl-pr myproc;
end-pr;

dcl-proc myproc;
end-proc;
