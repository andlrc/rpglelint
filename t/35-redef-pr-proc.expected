t/35-redef-pr-proc.rpgle:4:7: warning: redefinition of 'myint' as a different kind of symbol [-Wredefining-symbol]
 dcl-s myint int(10);
       ^
t/35-redef-pr-proc.rpgle:3:7: note: previous definition is here
 dcl-s myint int(10);
       ^
